x/dualstaking/types/types.go

```diff
@@ -1 +1,13 @@
 package types
+
+const (
+	DelegateEventName   = "delegate_to_provider"
+	UnbondingEventName  = "unbond_from_provider"
+	RedelegateEventName = "redelegate_between_providers"
+	RefundedEventName   = "refund_to_delegator"
+)
+
+const (
+	NotBondedPoolName = "not_bonded_dualstaking_pool"
```

Comment:
the uses for those are switched
bonded are for tokens that are delegated
unbonded are for tokens waiting to be returned to the delegator

Reply:
fixed 96a898ea